website/server/libs/taskManager.js

```diff
@@ -51,6 +51,10 @@ export async function createTasks (req, res, options = {}) {
     let taskType = taskData.type;
     let newTask = new Tasks[taskType](Tasks.Task.sanitize(taskData));
 
+    if (taskData.requiresApproval) {
```

Comment:
let's put this in the if (group) at line 60 so that it does nothing when passed to a task that is not part of a group. Also this parameter needs to be documented